tools/run_tests/run_performance_tests.py

```diff
@@ -435,7 +510,13 @@ def finish_qps_workers(jobs):
 if not args.dry_run:
   build_on_remote_hosts(remote_hosts, languages=[str(l) for l in languages], build_local=build_local)
 
-qpsworker_jobs = create_qpsworkers(languages, args.remote_worker_host)
+perf_cmd = None
+if args.perf_args:
+  # Expect /usr/bin/perf to be installed here, as is usual
+  perf_cmd = ['/usr/bin/perf'] 
```

Comment:
I had it at `/usr/bin/perf` onyl because the qps worker command is running with shell=False (so it doesn't get the PATH or find `perf` I take it, and what I saw).

Maybe qps worker commands should be changed to run using the shell?